pack/thor.json

```diff
@@ -1,4 +1,203 @@
 [
+	{
+		"attack": 2,
+		"code": "06001a",
+		"deck_limit": 1,
+		"defense": 2,
+		"faction_code": "hero",
+		"flavor": "\"Now you face the mightiest Avenger of all\"",
+		"hand_size": 4,
+		"health": 14,
+		"is_unique": true,
+		"name": "Thor",
+		"back_link": "06001b",
+		"pack_code": "thor",
+		"position": 1,
+		"quantity": 1,
+		"set_code": "thor",
+		"text": "\"Have at thee!\" — <b>Response</b>: After you engage a minion, draw 2 cards. (Limit once per phase.)",
+		"thwart": 1,
+		"traits": "Asgard. Avenger.",
+		"type_code": "hero"
+	},
+	{
+		"code": "06001b",
+		"deck_limit": 1,
+		"faction_code": "hero",
+		"hand_size": 5,
+		"health": 14,
+		"hidden": true,
+		"is_unique": true,
+		"name": "Odinson",
+		"pack_code": "thor",
+		"position": 1,
+		"quantity": 1,
+		"recover": 4,
+		"set_code": "thor",
+		"text": "Worthy — <b>Action</b>: Search your deck and discard pile for the Mjolnir upgrade and add it to your hand. Shuffle your deck. (Limit once per round).",
+		"traits": "Asgard.",
+		"type_code": "alter_ego"
+	},
+	{
+		"attack": 2,
+		"attack_cost": 1,
+		"code": "06002",
+		"cost": 4,
+		"deck_limit": 1,
+		"faction_code": "hero",
+		"flavor": "\"Where my Beloved sallies forth—let his Lady Sif be at his side!\"",
+		"health": 3,
+		"is_unique": true,
+		"name": "Lady Sif",
+		"pack_code": "thor",
+		"position": 2,
+		"quantity": 1,
+		"resource_wild": 1,
+		"set_code": "thor",
+		"set_position": 1,
+		"text": "<b>Response</b>: After Lady Sif enters play, ready Thor or Odinson.",
+		"thwart": 2,
+		"thwart_cost": 1,
+		"traits": "Asgard",
+		"type_code": "ally"
+	},
+	{
+		"code": "06003",
+		"cost": 0,
+		"deck_limit": 3,
+		"faction_code": "hero",
+		"name": "Defender of the Nine Realms",
+		"pack_code": "thor",
+		"position": 3,
+		"quantity": 2,
+		"resource_physical": 1,
+		"set_code": "thor",
+		"set_position": 2,
+		"text": "<b>Hero Action</b> <i>(thwart)</i>: Discard cards from the top of the encounter deck until you discard a minion. Put that minion into play engaged with you → remove 3 threats from a scheme.",
+		"traits": "Thwart.",
+		"type_code": "event"
+	},
+	{
+		"code": "06004",
+		"cost": 1,
+		"deck_limit": 1,
+		"faction_code": "hero",
+		"flavor": "\"For honor! For glory! For Asgard!\" -- Thor",
+		"name": "For Asgard!",
+		"pack_code": "thor",
+		"position": 4,
+		"quantity": 1,
+		"resource_physical": 1,
+		"set_code": "thor",
+		"set_position": 5,
+		"text": "<b>Alter-Ego Action</b>: Search your deck and discard pile for a card with the <b>ASGARD</b> trait and add it to your hand. Shuffle your deck.",
```

Comment:
For traits in card text we do the following
`[[Asgard]]`

And the renderer will take care of making it bold/italic